### ISSUE TITLE
result compare

### ISSUE THREAD

**Issue by KunWangV** [AceCoooool/DSS-pytorch#2]
Have you compared the results with paper?

**Comment by AceCoooool** [comment]
Sorry, I didn't compare with paper. (As I know, the author release the dataset now, you can train it by yourself )  --- I am not working on it recently, so i am sorry ～

**Comment by holyhao** [comment]
so，does anyone compare the result of this work with the paper？

**Comment by AceCoooool** [comment]
I will do it in October. （I am finding a job recently, so forgive me.）

**Comment by holyhao** [comment]
emm，i just want to make sure if this repo works well。

**Comment by AceCoooool** [comment]
I have update the code and results in pre-version，I will update results of v2（using learnable fusion） tomorrow

**Comment by holyhao** [comment]
> @holyhao I have update the code and results in pre-version，I will update results of v2（using learnable fusion） tomorrow

Thanks for your work. By the way,do you train the net on larger batchsize and lr. I see that you set batchszie only 8 and lr is even small.

**Comment by AceCoooool** [comment]
1. I think it's better to use larger lr (I find the loss curve decrease too slow at the beginning). --- but I did not try it (the learning in the paper is very small --- 1e-8)
2. You can try larger batchsize（and can also use vgg with batch normalization --- I use pre-trained vgg without bn layer, and use a pre-trained model from caffe may increase the results, many projects find caffe-pretrained model have good performance）. 
welcome to share your results, thank you ~

**Comment by holyhao** [comment]
I try lr=1e-4 with backbone resnet18 ,it works fine and converges faster. But when i try larger batchsize like 32,48, it converges slow and gets worse val results. It confused me, as far as I known, larger batchsize should leads to better results. Do you have some ideas about this.

**Comment by AceCoooool** [comment]
I did not have a machine with large memory（GPU）, so i have no "engineering experiment" in large batch size. However, there is several discuss about bach-size:
1. [stack exchange](https://stats.stackexchange.com/questions/164876/tradeoff-batch-size-vs-number-of-iterations-to-train-a-neural-network)
2. 《Deep Learning》Ian p172 (Chinese version)：small batch size may have regularization, with better generalization.

However, I think there must be some practical tip to train model in larger batch size. （I am sorry without good suggestions. :cry: ）

**Comment by holyhao** [comment]
Your reply  really inspires me and thank you very much.
